src/io/XyzWriter.cpp

```diff
@@ -18,23 +18,23 @@ using namespace std;
 void XyzWriter::readXML(XMLfileUnits& xmlconfig) {
 	_writeFrequency = 1;
 	xmlconfig.getNodeValue("writefrequency", _writeFrequency);
-	global_log->info() << "Write frequency: " << _writeFrequency << endl;
+	global_log->info() << "Write frequency: " << _writeFrequency << std::endl;
```

Comment:
why not replace those `endl` too while we are at it?

Reply:
for the log it is sometimes useful, if everything else fails afterwards, s.t., the last state is written.
But yes, one could replace that, too.